### ISSUE TITLE
Selecting a mention from keyboard does not select the mention

### ISSUE THREAD

**Issue by khawerrind** [draft-js-plugins/draft-js-plugins#766]
Thank you for the great plugins. When i try to select a mention from the suggestion list using keyboard and press enter the mention does not gets selected instead i get a new line in the editor. However selecting a mention using mouse works perfectly.
I also tried to return `handled` for the `handleReturn` function of the Editor but that didn't help as well. Any suggestions for this issue?

```
handleEditorReturn() {
        return 'handled';
}

<Editor
        editorState={this.state.editorState}
        onChange={this.onEditorStateChange}
        plugins={plugins}
        ref={(element) => { this.editor = element; }}
        handleReturn={this.handleEditorReturn}
/>
```

I am using the following versions

```
        "draft-js": "^0.10.1",
        "draft-js-mention-plugin": "^2.0.0-rc2",
        "draft-js-plugins-editor": "^2.0.0-beta1",
```
I tried to console the `value` of `isEventHandled` function and the value is coming as false
```
function isEventHandled(value) {
        console.log(value); //it says false  
        return value === 'handled' || value === true;
}
```

**Comment by icd2k3** [comment]
Having the same issue here, but using `handleKeyCommand` - also when I hover my mouse over the menu the highlight doesn't change.

**Comment by rossfishkind** [comment]
Strangely enough, I got this working by adding a state for `editorState` on the component that displays my editor.

During the `onChange` function of `Editor` I added:

```
this.setState({
  editorState,
});
```

It had been code I was using during a separate test and had forgotten to remove, but I found that with it there, regardless that I don't use `this.state.editorState` anywhere, it makes the keyboard selection work. I hope this helps someone.
